src/components/cc-ssh-key-list/cc-ssh-key-list.js

```diff
@@ -141,6 +149,33 @@ export class CcSshKeyList extends LitElement {
     }
   }
 
+  _isPublicKeyPrivate (publicKey) {
+    const lowerCasePublicKey = publicKey.toLowerCase();
+    return lowerCasePublicKey.match(' private ') || lowerCasePublicKey === 'private';
+  }
```

Comment:
This is a bit funny because, the whole point was to detect ` private ` surrounded by spaces to identify a real private key but now because of the trim thing, we also detect just the keyword `private`.

It's not really a problem but maybe we could simplify this a bit.

Reply:
Maybe we should call this function with the untrimmed value?

Reply:
> Maybe we should call this function with the untrimmed value?

Good idea! I'll give it a try!